infra/resources/_modules/monitoring/monitoring.tf

```diff
@@ -0,0 +1,115 @@
+data "azurerm_application_insights" "common" {
+  name                = "io-p-ai-common"
+  resource_group_name = "io-p-rg-common"
+}
+
+resource "azurerm_monitor_action_group" "io_com_error" {
+  name                = "io-p-com-error-ag-01"
+  resource_group_name = var.resource_group_name
+  short_name          = "iocom-error"
+
+  email_receiver {
+    name                    = "slack"
+    email_address           = var.io_com_slack_email
+    use_common_alert_schema = true
+  }
+}
+
+resource "azurerm_monitor_scheduled_query_rules_alert" "message-ingestion-alert" {
+  name                = format("[%s] Message Ingestion Error", var.project)
+  resource_group_name = var.resource_group_name
+  location            = var.location
+
+  data_source_id          = data.azurerm_application_insights.common.id
+  description             = "[IO-COM] Alerts for all the errors during the message ingestion to data lake"
```

Comment:
Just a a consideration, maybe we can replace `[IO-COM]` with something related to the ingestion now that we have a dedicated slack channel for the domain.